salt/metalk8s/orchestrate/deploy_new_node.sls

```diff
@@ -0,0 +1,74 @@
+{%- if pillar['bootstrap_id'] %}
+{%-   set control_plane_ips = salt.saltutil.runner('mine.get', tgt=pillar['bootstrap_id'], fun='control_plane_ip') %}
+{%- else %}
+{%-   set control_plane_ips = {} %}
+{%- endif %}
+
+{%- if pillar['bootstrap_id'] in control_plane_ips.keys() and control_plane_ips[pillar['bootstrap_id']] %}
+{%-   set control_plane_ip = control_plane_ips[pillar['bootstrap_id']] %}
+{%- else %}
+{%-   set control_plane_ip = 'localhost' %}
```

Comment:
Isn't this an error? A 'new node' is, quite by definition, not the bootstrap node, so configuring it to go to `localhost` for Salt master, repositories,... will... almost certainly not work.

Reply:
Just a copy/paste of previous orchestrate lets fix this in #1017

Reply:
OK